tensorflow/contrib/crf/python/kernel_tests/crf_test.py

```diff
@@ -199,6 +200,87 @@ def testViterbiDecode(self):
       self.assertEqual(actual_max_sequence,
                        expected_max_sequence[:sequence_lengths])
 
+  def testCrfDecode(self):
+
+    def np_crf_decode(potentials, transition_params, seq_lens):
+      '''This is a function for numpy arrays.
+
+      Args:
+        potentials: [B, T, num_tags]
+        transition_params: [num_tags, num_tags]
+        seq_lens: [B]
+        tag_indices: [B, T]
+
+      Returns:
+        pred_tags: a list
+        scores: a list
+      '''
+      pred_tags, scores = [], []
+      for p, seq_len in zip(potentials, seq_lens):
+        p = p[:seq_len]
+        pred_tag, score = crf.viterbi_decode(p, transition_params)
+        pred_tags.append(pred_tag)
+        scores.append(score)
+      return pred_tags, scores
+
+
+    def calc_accuracy(pred_tags, gold_tags, seq_lens):
```

Comment:
That's fine. Usually, I like to have a toy, deterministic example that can be worked out by hand. Would you mind adding that?